Dockerfile

```diff
@@ -11,9 +14,17 @@ COPY Gemfile Gemfile
 COPY Gemfile.lock Gemfile.lock
 
 RUN gem update --system
-RUN bundle config --local without test:development && bundle install
-
-
+# Gems installed (includes compilation)
+RUN bundle config --local without test:development && \
+    bundle install && \
+    # remove gem cache
+    rm -rf /usr/local/bundle/cache && \
+    # fix permissions of files that were found to be world writable
+    chmod -R o-w /usr/local/bundle/gems/os-1.1.4
```

Comment:
would it be good to just do this on the whole bundle rather than this one specific gem?

Reply:
That would be good to catch future things that we install that needed this treatment. I tested the speed, and on my machine the dockerfile build takes 0.3s for `usr/local/bundle/gems/os-1.1.4` or 9.2s for `usr/local/bundle/`. So in the scheme of things, not much, so I'll adopt this 👍